### ISSUE TITLE
Is Proxy actually working?

### ISSUE THREAD

**Issue by ChrisJSt** [instagram4j/instagram4j#88]
Hi, how do I use a proxy when logging into Instagram using this API? I have used the code mentioned from a previous post:

```
Instagram4j instagram = Instagram4j.builder().username(userName).password(password).build();
        instagram.setup();
            HttpHost proxy = new HttpHost("IP_SERVER", PORT_NUMBER, "http");
            instagram.getClient().getParams().setParameter(ConnRoutePNames.DEFAULT_PROXY, proxy);
            instagram.getClient().getParams().setIntParameter("http.connection.timeout", 600000);

            instagram.getClient().getCredentialsProvider().setCredentials(
                    new AuthScope("IP_SERVER", PORT_NUMBER),
                    new UsernamePasswordCredentials(NETWORK_USER, NETWORK_PASS));
        try {
            instagram.login();
        } catch (Exception ex) {
            ex.printStackTrace();
        }
```

However, when I test a foreign proxy on my account, I don't receive any checkpoint required upon logging  in, which I expect to receive if the proxy is in fact working. Any help?

**Comment by rasoolta** [comment]
hi chrisjstelmach
please hep me
How did you sync this libarary (instagram4j)?
I try to build this lib in android stadio, But I did not succeed 😞😞
pease help me to build this lib
can you send gradle file for me
please help ....
my mail is rasoolta737@gmail.com

**Comment by patriotaki** [comment]
how can you use proxy without authentication user/pass? just use ip and port? any code example? thx

**Comment by DzianisH** [comment]
Hi All,
Proxy will be available in 1.8 version.
```java
        Instagram4j instagram = Instagram4j.builder()
                .username("your email")
                .password("your password")
                .proxy(new HttpHost("host.address.com", 8080, "http"))
                .build();

        instagram.setup();
        instagram.login();

        InstagramLikeRequest request = new InstagramLikeRequest(InstagramCodeUtil.fromCode("BfjRJp1nQug"));
        InstagramLikeResult result = instagram.sendRequest(request);
        System.out.println(result);
```
